rest/src/test/java/discord4j/rest/service/UserServiceTest.java

```diff
@@ -63,7 +63,7 @@ public void testGetInvalidUser() {
         try {
             getUserService().getUser(1111222).block(); // should throw ClientException
         } catch (ClientException e) {
-            ErrorResponse response = e.getErrorResponse();
+            ErrorResponse response = e.getErrorResponse().orElse(null);
```

Comment:
It seems weird to have a nullable variable used just after. I know the previous behaviour could also throw a NPE but maybe add a `if(response != null) { ... } else { ... }` ?

Reply:
What about throwing a `NoSuchElementException`?

Reply:
Yes, you could use `.orElseThrow()` because it's not really intended to have an empty optional here I think

Reply:
Yeah to be fair the current example is also prone to that NPE if response is null. My suggestion is to replace the catch block with a `toString` of `ClientException`. That means you need to change that method too in this PR.

Reply:
So just put `System.out.println("Error: " + e.toString());` in the catch block?

Reply:
As long as toString behaves appropriately on ErrorResponse == null case